force-app/main/default/classes/RegionManagerController.cls

```diff
@@ -0,0 +1,33 @@
+public with sharing class RegionManagerController {
+  @AuraEnabled(cacheable=true)
+  public static Communal_Service_User__c getRegionManagerByUserId(
+    String userId
+  ) {
+    try {
+      return [
+        SELECT Id, User__c, Region_Manager__r.Email__c, Email__c
+        FROM Communal_Service_User__c
+        WHERE User__c = :userId
+        LIMIT 1
+      ];
+    } catch (Exception e) {
+      throw new CustomException(e.getMessage());
+    }
+  }
+
+  @AuraEnabled
+  public static void contactManager(String payload) {
+    try {
+      Map<String, Object> payloadMap = (Map<String, Object>) JSON.deserializeUntyped(
```

Comment:
Do you expect any non-String values in this map? Map<String, Object> or Map<String,String>?